app/models/user.js

```diff
@@ -40,4 +43,21 @@ userSchema.plugin(passportLocalMongoose, {
     }
 })
 
+async function getLink(link, docType, docId, n = 1) {
```

Comment:
Looks like you have duplicated `getLink` function

Reply:
Fixed